Analytics/Defaults.cs

```diff
@@ -1,22 +1,28 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using Segment.Model;
 
 namespace Segment
 {
     public class Defaults
     {
-        public static string Host = "https://api.segment.io";
+        public static readonly string Host = "https://api.segment.io";
```

Comment:
Why did you change these to `readonly`?

Reply:
Because those are the defaults value and I think that the final user shouldn't be able to change it.

Reply:
I assume we will be able to programmatically change it when we instantiate a new analytics object? For example, we might want to change the host to something else during initialization/configuration.